src/resources/measurements.ts

```diff
@@ -12,13 +63,24 @@ class Measurements {
   /**
    * Send Measurement (one or multiple).
    */
-  public create(body: Measurement | Array<Measurement>, options?: any) {
+  public create(body: Measurement): Promise<Measurement>;
+  public create(body: Measurement[]): Promise<Measurement[]>;
+  public create(body: MeasurementInput): Promise<Measurement>;
+  public create(body: MeasurementInput[]): Promise<Measurement[]>;
+  public create(
+    body: MeasurementInput | MeasurementInput[],
+    options?: any,
+  ): Promise<Measurement> | Promise<Measurement[]> {
     if (body instanceof Array) {
```

Comment:
this typeguards properly right? Asking for my own understanding 😆

Reply:
haha yeah! In this `if` block, `body` is `MeasurementInput[]`. TS seems ok with my function implementation using `MeasurementInput` as the type for `body` (rather than something like `MeasurementInput | Measurement`) since Measurement is assignable to MeasurementInput.